config/settings.yml

```
@@ -35,4 +35,8 @@ discourse_activity_pub:
   activity_pub_verbose_logging:
     default: false
   activity_pub_object_logging:
-    default: false
\ No newline at end of file
+    default: false
+  activity_pub_logs_max_days_old:
+    default: 60
+    min: 7
+    max: 36500
```

```suggestion
    min: 2
    max: 365
```

Nitpick, if a site has too much activity, they might want cleanup more often. And max beyond a year seems unnecessary.